tests/cli/bootprep/input/test_configuration.py

```diff
@@ -364,6 +406,16 @@ def test_branch_property_not_present(self):
         """Test the branch property when branch is not in the layer data"""
         self.assertIsNone(self.version_layer.branch)
 
+    def test_branch_property_jinja_template(self):
+        """Test the branch property when the branch uses Jinja2 templating"""
+        # Have to double the literal brackets that make up the Jinja2 variable reference
+        self.branch_layer_data['product']['branch'] = f'integration-{{{{{self.product_name}.version}}}}'
```

Comment:
Same question as above, is this more readable?
```suggestion
        self.version_layer_data['product']['branch'] = 'integration-{{' + f'{self.product_name}.branch' + '}}'
```